### DISCUSSION TITLE
Show case insensitive results when searching for a branch

### DISCUSSION THREAD

**Discussion by Asaf-S** [community/community#59061] [Repositories]
### Select Topic Area

Product Feedback

### Body

Whether in the main repo screen or when creating a pull request from your UI, the branches are filtered according to the inputted text in a case-sensitive way when searching for a branch.
This is especially helpful for people with multiple languages in their OS, and who are currently using a different language and use the `SHIFT + [LETTER]` or `CAPS LOCK` to quickly input the branch name.

I agree that there is a difference in the cases of branches, however, this should not be applied to the filter, as if I'm searching for the text `Us-1`, it will show me both the `us-1` and `US-1` results - and I'll have to choose between them.

**Comment by nrdevau** [comment]
Can anyone confirm this is the right place for Feature Requests in the Github web UI?

Ran into this issue 3 or four times in the past week. While enforcing correct branch names when creating and checking out branches, I think comparing branches could be a little bit less strict in the branch name search. Even some sort of fuzzy searching would help
